### ISSUE TITLE
[Internal] Fix build issue

### ISSUE THREAD

**Issue by samtstern** [firebase/FirebaseUI-Android#1786]
#1774 and #1785 both hit this issue:
```
> Task :proguard-tests:lint FAILED
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':proguard-tests:lint'.
> Could not resolve all dependencies for configuration ':auth:debugRuntimeClasspath'.
   > A problem occurred configuring project ':internal:lintchecks'.
      > Cannot change strategy of configuration ':internal:lintchecks:classpath' after it has been resolved.
```

We need to fix this issue before we can safely merge any more code.

cc @puf  @rosariopfernandes

**Comment by samtstern** [comment]
Note: this was caused by `configureondemand`

**Comment by samtstern** [comment]
Fixed in #1789
